redux/apis/api-types.ts

```diff
@@ -0,0 +1,67 @@
+export interface ErrorResponse {
+  data: {
+    message: string;
+    name: string;
+    status: number;
+    response: {
+      error: string;
+      message: string;
+      statusCode: number;
+    };
+  };
+  status: number;
+}
+
+export interface SignUpRequest {
+  email: string;
+  password: string;
+  confirmPassword: string;
+  local: boolean;
+}
+
+export interface SignUpSuccessResponse {
+  data: {
+    message: string;
+    data: {
+      isOrg: boolean;
+    };
+  };
+}
+
+export interface ResendEmailVerificationLinkRequest {
+  email: string;
+  local: boolean;
+}
+
+export interface ResendEmailVerificationLinkResponse {
+  data: {
+    message: string;
+    data: string;
+  };
+}
+
+export interface VerifyEmailRequest {
```

Comment:
You can directly use a string where you are currently using this type. There isn't much available here